helix-term/src/commands/typed.rs

```diff
@@ -2051,6 +2101,13 @@ pub const TYPABLE_COMMAND_LIST: &[TypableCommand] = &[
             fun: reload,
             completer: None,
         },
+        TypableCommand {
+            name: "reload-all",
+            aliases: &["ra"],
```

Comment:
I don't think this command needs an alias. `ra` might be confused for some sort of `:read-all` command similar to `:read` (which is in a PR)

Reply:
sure, I'll remove this alias.